lib/simpler/controller.rb

```diff
@@ -16,14 +16,27 @@ def make_response(action)
       @request.env['simpler.action'] = action
 
       set_default_headers
+      get_params
+
       send(action)
+      
       write_response
 
       @response.finish
     end
 
     private
 
+    def get_params
+      params.merge!(@request.env['params_stash'])
+    end
+
+    def set_custom_headers(headers)
+      headers.each do |header|
```

Comment:
В принципе у нас не было задачи иметь возможность устанавливать несколько заголовков разом. Но ок, пусть будет. Только проще сразу разбить на параметры
```suggestion
      headers.each do |name, value|
```